test/actions/createMetadataAndME.test.ts

```diff
@@ -74,5 +72,5 @@ describe('creatomg metadata and master edition PDAs', () => {
     const editionInfo = await Account.getInfo(connection, edition);
     const deserializedEditionData = new MasterEdition(edition, editionInfo).data;
     expect(deserializedEditionData.maxSupply.toString(10)).toEqual('100');
-  });
+  }, 150000);
```

Comment:
I'd rather move these to constants and try to reuse some of those in other tests in the future

Reply:
Transaction complexity VS Time

Reply:
Obviously, the timeout depends on async operations count + network conditions. So they are will be different each time.